### ISSUE TITLE
Customize BibTeX key

### ISSUE THREAD

**Issue by domoritz** [citation-file-format/citation-file-format#407]

**Comment by jspaaks** [comment]
My two cents: there is no good way to automatically generate a key. Even if authors supply it, consumers might prefer a key in a different format. Hence the `YourReferenceHere`.

**Comment by hainesr** [comment]
I agree @jspaaks. Also, the reason we moved from `YourReferenceHere` to a generated citekey was to help people who needed to copy more than one CFF generated reference into a `.bib` file. The current citekeys should rarely clash at least.
